### PR TITLE
Dup - Microsoft.XNARedist version 4.0.20823

### PR DESCRIPTION
**Pull request by Masamune3210**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27271)

### CODE REVIEW

File: manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.XNARedist
+PackageVersion: 4.0.20823
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.0.20823.0
```

File: manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.XNARedist
+PackageVersion: 4.0.20823
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.installer.yaml:5]:
```suggestion
PackageVersion: 4.0.20823.0
```

File: manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.XNARedist
+PackageVersion: 4.0.20823
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.yaml:5]:
```suggestion
PackageVersion: 4.0.20823.0
```

Comment (ItzLevvie) [reply manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.yaml:5]: `PackageVersion` must match Control Panel. You'll also have to update the folder from `4.0.20823` to `4.0.20823.0`

File: manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.installer.yaml:5]: `PackageVersion` must match Control Panel. You'll also have to update the folder from `4.0.20823` to `4.0.20823.0`

File: manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.locale.en-US.yaml:5]: `PackageVersion` must match Control Panel.

File: manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.XNARedist
+PackageVersion: 4.0.20823
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft XNA Framework Redistributable
+License: Copyright Microsoft
+ShortDescription: XNA Framework Redist for software that requires it
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.locale.en-US.yaml:10]:
```suggestion
ShortDescription: XNA Framework Redistributable download provides game developers with the XNA Framework run-time libraries they can include with their product for redistribution on the Windows platform.
```

File: manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.XNARedist
+PackageVersion: 4.0.20823
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft XNA Framework Redistributable
+License: Copyright Microsoft
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.locale.en-US.yaml:9]:
```suggestion
PackageUrl: https://www.microsoft.com/en-us/download/details.aspx?id=20914
License: Proprietary
Copyright: (c) Microsoft Corporation. All rights reserved.
CopyrightUrl: https://www.microsoft.com/en-us/legal/terms-of-use
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.yaml (hunk 3 above)
Comment (Masamune3210) [reply manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.yaml:5]: Was unsure, now I know thanks

**Pull request closed without merging**